public/manager-styles.css

```diff
@@ -136,3 +136,27 @@ button[title="Clear search"], button[title="Clear search"]:hover {
         height: calc(100vh - 140px);
     }
 }
+
+
+/* Fixing https://github.com/microsoft/FluidFramework/issues/3815 */
```

Comment:
nit: I read this as "the fix for #3815 is in the selector _immediately_ below" but it's really lines 142-162. Probably not worth addressing. Is there a `#region` convention in CSS?

Reply:
i made the block of fixes a bit more obvious